tests/api_test/api_test.cpp

```diff
@@ -1228,4 +1229,72 @@ TEST_CASE("ioctl_stress", "[stress]")
     bpf_object__close(object);
 }
 
-#endif
\ No newline at end of file
+TEST_CASE("test_ringbuffer_wraparound", "[stress]")
+{
+    // Load bindmonitor_ringbuf.sys
+    struct bpf_object* object = nullptr;
+    fd_t program_fd;
+    uint32_t event_count = 0;
+    std::string app_id = "api_test.exe";
+    uint32_t thread_count = 2;
+    native_module_helper_t native_helper;
+    native_helper.initialize("bindmonitor_ringbuf", EBPF_EXECUTION_NATIVE);
+
+    REQUIRE(
+        _program_load_helper(
+            native_helper.get_file_name().c_str(), BPF_PROG_TYPE_BIND, EBPF_EXECUTION_NATIVE, &object, &program_fd) ==
+        0);
+
+    // Get fd of process_map map
+    fd_t process_map_fd = bpf_object__find_map_fd_by_name(object, "process_map");
+
+    uint32_t max_entries = bpf_map__max_entries(bpf_object__find_map_by_name(object, "process_map"));
+    uint32_t max_iterations = static_cast<uint32_t>(10 * (max_entries / app_id.size()));
+    printf("max_iterations: %d\n", max_iterations);
+    REQUIRE(max_iterations % thread_count == 0);
+    uint32_t iterations_per_thread = max_iterations / thread_count;
+
+    // Subscribe to the ring buffer.
```

Comment:
nit: some comments end in a period and others don't.  Would be nice to be consistent throughout this file.

Reply:
Fixed the comments in the new test.